addons/test_mail/models/test_mail_thread_models.py

```diff
@@ -10,3 +10,21 @@ class MailTestCC(models.Model):
     _inherit = ['mail.thread.cc']
 
     name = fields.Char()
+
+
+class MailTestTrackingValue(models.Model):
```

Comment:
You may define it in test_mail_corner_case_models.py (badly named I know). This one is more for thread overrides (mail.thread.cc and I plan to improve a bit other models one day or another).